.github/workflows/superset-frontend.yml

```diff
@@ -143,7 +143,7 @@ jobs:
       - name: tsc
         run: |
           docker run --rm $TAG bash -c \
-          "npm run type"
+          "npm run plugins:build && npm run type"
```

Comment:
Do we NEED to do the plugins:build here? I think it's good hygeine, and we should be doing this in pre-commit as well, but there are some concerns about it being too slow for pre-commit.

Reply:
We're hoping that `tsc` is sufficient for the plugins/packages :D

Reply:
Sorry to keep rambing, but if we DO need it, we might want to try pnpm/bun here to speed things up quite a bit.

Reply:
@rusackas I think correctness beats slowness in this case as if a plugin type changed, we need to rebuild them in order for `npm run type` to work. If you know a faster way of doing that while generating the types when needed, it would be awesome.